virttest/utils_test/libvirt_device_utils.py

```diff
@@ -1,14 +1,14 @@
-import os
 import uuid
 
 from virttest.libvirt_xml.devices.filesystem import Filesystem
 
 
-def create_fs_xml(fsdev_dict):
+def create_fs_xml(fsdev_dict, launched_mode):
```

Comment:
Do you want to set launched_mode as one default value:auto?

Reply:
Same concern here, do we need default value here?

Reply:
Modified.